src/pages/workspaces/workspace/WorkspaceTabs.test.ts

```diff
@@ -135,8 +149,10 @@ describe('WorkspaceTabs', () => {
       setLeavingWorkspace: () => {},
       refresh: () => {},
     };
+
     // Act
     render(h(WorkspaceTabs, props));
+    delay(5);
```

Comment:
Why is this delay necessary? I'm wonder if you can flush promises, advance a timer, etc. to avoid it. If it has to stay, a comment should be added to explain why it is needed.

Reply:
That was from debugging mocks yesterday. I thought I took it out, but it snuck back in from a stashed change.

Reply:
It seems especially dubious since it's not awaited.